router/middleware/logger.go

```diff
@@ -20,6 +20,33 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// This file, in part, reproduces portions of
+// https://github.com/elastic/ecs-logging-go-logrus/blob/v1.0.0/formatter.go
+// to handle our custom fields in Format().
+
+const (
+	// ecsVersion holds the version of ECS with which the formatter is compatible.
+	ecsVersion = "1.6.0"
+)
+
+var (
+	ecsFieldMap = logrus.FieldMap{
+		logrus.FieldKeyTime:  "@timestamp",
+		logrus.FieldKeyMsg:   "message",
+		logrus.FieldKeyLevel: "log.level",
+	}
+)
```

Comment:
instead of having these global variables, let's move them inside the .Format method. We don't need them anywhere else.

Reply:
addressed